tests/test_live.py

```diff
@@ -87,7 +87,7 @@ def test_instruments():
         host=get_platform_dispatcher(),
         instrument="mock",
     )
-    assert disp.get_instruments_list() == ['isgri', 'jemx', 'polar', 'antares', 'spi_acs']
+    assert disp.get_instruments_list() == ['isgri', 'jemx', 'polar', 'gw', 'spi_acs', 'legacysurvey']
```

Comment:
would have been better to always sort them

Reply:
Yes, will do with the next PR.